man/docker-network-inspect.1.md

```diff
@@ -0,0 +1,52 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% OCT 2015
+# NAME
+docker-network-inspect - inspect a network
+
+# SYNOPSIS
+**docker network inspect INSPECT**
```

Comment:
I haven't got to the man pages yet...sorry this PR is a work in progress and not really ready for a formal review.